pkg/ranchhand/rancher.go

```diff
@@ -37,7 +37,8 @@ var (
 				Namespace: "cattle-system",
 				Version:   "2019.3.1",
 				SetValues: map[string]string{
-					"tls": "external",
+					"tls":       "external",
+					"privateCA": "true",
```

Comment:
I'm struggling to understand why we need to set `"privateCA": "true"` here. 

By virtue of setting TLS to `external`, we've essentially configured this cluster so that the Nginx ingress controller will handle TLS. Creating the default certificate for that component with the proper CA and CommonName should be adequate, no?

Reply:
In my testing, I was finding that the ca-checksum being created was incorrect so I'm importing it into rancher as a private ca to ensure rancher's canonical ca cert matches the one we're using as an endpoint.